loki-client/src/main/java/com/github/loki4j/client/pipeline/PipelineConfig.java

```diff
@@ -187,6 +197,17 @@ public static class Builder {
         private long sendQueueMaxBytes = batchMaxBytes * 10;
         private int maxRetries = 2;
         private long retryTimeoutMs = 60 * 1000;
+        private Supplier<Long> jitter = new Jitter();
+        private BiFunction<Integer, Long, Boolean> sleep = (attempt, timeout) -> {
+            try {
+                long backoff = timeout * (2 << (attempt - 1));
```

Comment:
Shouldn't `backoff` be equal to `timeout` on the first retry?

Reply:
Changed.